addons/fleet/models/fleet_vehicle.py

```diff
@@ -17,7 +17,7 @@ def _get_default_state(self):
         return state and state.id or False
 
     name = fields.Char(compute="_compute_vehicle_name", store=True)
-    active = fields.Boolean(default=True)
+    active = fields.Boolean('Active', default=True, track_visibility="onchange")
```

Comment:
Could you check if the email is sent when the record is inactive? I have a doubt.